include/lsst/sphgeom/Chunker.h

```diff
@@ -48,8 +48,8 @@ namespace sphgeom {
 ///
 /// TODO(smm): implement a more memory efficient representation than this.
 struct SubChunks {
-    int32_t chunkId;
-    std::vector<int32_t> subChunkIds;
+    std::int32_t chunkId;
```

Comment:
Maybe this file should be including `<cstdint>` and not `<stdint.h>`? (not sure if there are other cases of this).

Reply:
```$ rg stdint.h sphgeom
sphgeom/include/lsst/sphgeom/BigInteger.h
36:#include <stdint.h>

sphgeom/include/lsst/sphgeom/Chunker.h
37:#include <stdint.h>

sphgeom/tests/test.h
36:#include <stdint.h>```